spec/models/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+  subject { create(:topic) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_uniqueness_of(:name) }
+  end
+
+  describe '#create' do
```

Comment:
As it is a Class method and not a instance method it should be `describe '.create' do `

Reply:
Thanks, you are right.